src/sources/core.js

```diff
@@ -128,6 +175,78 @@ class CoreDataset extends Dataset {
   }
 }
 
+class CoreCollectedResources extends CollectedResources {
+
+  static objectName(object) {
+    return coreBucketKeyMunger(object);
+  }
+
+  lastUpdated(version=this._versions[0]) {
+    return version.LastModifiedDate;
+  }
+  
+  filter(options) {
+    /**
+     * We can perform authorization here by either calling 
+     * authz.authorized(user, authz.actions.Read, object),
+     * See note above for more details.
+     */
```

Comment:
@tsibley threading the authz conversation here. For our other APIs, the authz check is done on a single resource (`Resource` subclass instance). Here we have a _collection_ of resources, which may include resources with different filenames etc.

We can easily perform an authorization check on the `CollectedResources` instance, but to stay properly aligned with the current approach the check should potentially be on each individual resource within the collection.